manifests/public.yaml

```diff
@@ -181,9 +178,6 @@ spec:
     - mountPath: /etc/pki/ca.crt
       name: ca-certificate
       readOnly: True
-    - mountPath: /etc/salt/master.d/50-master.conf
-      name: salt-master-config-master-conf
-      readOnly: True
```

Comment:
> i can make a testrun,

ok, `/setup/discovery` immediately throws `Velum::SaltApi::SaltConnectionException` I guess it's because of this change